elderly_assistant/services/http_client.py

```
@@ -52,11 +60,29 @@ def __init__(self, config):
         self.device_token = _load_device_token()
 
+    def _reload_device_token(self):
+        """从持久化文件重新读取 device_token，覆盖内存中可能过期的令牌。
+
+        场景：配网流程（wifi_config.register_device_to_server）或服务端其他路径
+        已将最新令牌写入文件，而本 HTTPClient 实例在构造时令牌尚为空/旧，
+        若不重载，将一直携带空/旧 X-Device-Token，导致鉴权端点永久返回 403。
+        """
+        token = _load_device_token()
+        if token and token != self.device_token:
+            logger.info("从本地文件重新加载 device_token（覆盖内存中过期的令牌）")
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)